Updated API Key Best Practices and Examples setup to use Okta CLI (#650)
oktadeveloper/okta-blog: _source/_posts/2021-02-03-api-key-best-practices-and-examples.md

## How to Obtain and Use an Okta API Token

{% include setup/cli.md type="spa" loginRedirectUri="http://localhost:8080" %}

> Next, create an API token. Run `okta login` and open the resulting URL in your browser. Log in and go to **Security** > **API** > **Tokens**. Create a new token and store the value somewhere safe. Make sure you don't check it into GitHub! 

Reviewer: @Dr-Phill-Edwards This prompted me to add a new `type="token"` to the CLI include. Now, you can replace these two paragraphs with:

```
{% include setup/cli.md type="token" %}
```

You can see what the rendered HTML looks like at http://developer.okta.com/blog/setup#api-token.

When making an API call, the token needs to be added in an Authorization HTTP request header. The token type must be `SSWS`, which is the proprietary authentication scheme used by Okta. Replace `00...3` with the actual token.

```http
Authorization: SSWS 00...3
```

> An API call using the API token can be made using `curl`. Replace `${OKTA_API_KEY}` with the API token and replace `${OKTA_DOMAIN}` with the domain obtained from the developer console.

Reviewer: ```suggestion
An API call using the API token can be made using `curl`. Replace `${OKTA_API_KEY}` with the API token and replace `${OKTA_DOMAIN}` with your Okta domain.
```
